Lib/defconQt/fontInfo.py

```diff
@@ -790,14 +785,14 @@ class PostScriptTab(TabWidget):
     def __init__(self, font, parent=None):
         super().__init__(parent, name="PostScript")
```

Comment:
I tried this at first, but get `RuntimeError: super-class __init__() of type GeneralTab was never called`.